olla/include/openmm/Platform.h

```diff
@@ -35,6 +35,7 @@
 #include <map>
 #include <string>
 #include <vector>
+#include <sstream>
```

Comment:
This include isn't actually needed anywhere in the header.  So better to move it to Platform.cpp.  That way it will only get included in that one file, not in everything that includes this header.

Reply:
I couldn't get it to compile without adding it there, though I haven't the foggiest idea why (`message` later on is a stringstream, too, and caused no problems). It could be specific to my build system, or maybe I had something misconfigured. I'll remove it and see if, in a completely clean env, it's still required.

Reply:
How strange.  You do need to include that header, but you should be able to do it in the .cpp file rather than the .h file.

Reply:
Ooooh I see the problem: the sstream import was inside ONLY the `#ifdef WIN32` block.

Reply:
Moved it outside the ifdef block, compiles fine now.